src/test/java/com/karankumar/bookproject/ui/book/BookFormTest.java

```diff
@@ -97,38 +103,49 @@ public void setup() {
         customShelfService.save(customShelf);
 
         Assumptions.assumeTrue(predefinedShelfService != null);
-        bookForm = createBookForm(true);
-
+        bookForm = createBookForm(READ, true);
     }
 
-    private BookForm createBookForm(boolean isInSeries) {
+    private BookForm createBookForm(ShelfName shelf, boolean isInSeries) {
```

Comment:
Instead of passing in the ShelfName, what about passing in a `PredefinedShelf.ShelfName` instead?

Reply:
I can do that, but can you explain to me which benefit that has?
Both do point to the same enum, however using only ShelfName it is easier to read imo.

Reply:
Ignore me, I was confusing this with the `Shelf` class

Reply:
done
(But i still do not understand the benefit)

Reply:
So this was a mistake on my part. You can ignore the suggestion and go back to using `ShelfName`.